src/Compilers/CSharp/Portable/FlowAnalysis/CSharpDataFlowAnalysis.cs

```diff
@@ -292,5 +294,13 @@ public sealed override bool Succeeded
                 return _succeeded.Value;
             }
         }
+
+        internal static ImmutableArray<ISymbol> Sort(IEnumerable<Symbol> data)
+        {
+            var builder = ArrayBuilder<Symbol>.GetInstance();
+            builder.AddRange(data);
+            builder.Sort(LexicalOrderSymbolComparer.Instance);
+            return builder.ToImmutableAndFree().As<ISymbol>();
```

Comment:
Is it possible that `data` contains 2 local variables with the same name declared in disjoint blocks?